packages/zoe/src/zoe.js

```diff
@@ -212,14 +223,23 @@ import { makeTables } from './state';
  * queries based on other information, we choose to omit it. For
  * instance, `installationHandle` can be derived from
  * `instanceHandle` and is omitted even though it is useful.
- * @param  {object} seat - an object defined by the smart
+ * @param  {OfferHook} offerHook - a function that will be handed the seat at
```

Comment:
There are no seats.

"...handed the `offerHandle` at..."

Reply:
addressed in 43aa15b1